Packs/CommonScripts/ReleaseNotes/1_19_57.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### ContentPackInstaller
+
+- Updated the ContentPackInstaller script to use server URL when installing a pack for XSIAM or XSOAR 8.X and above.
```

Comment:
Updated the ContentPackInstaller script to use the server URL when installing a pack for XSIAM or XSOAR 8.X and above.